doc/gov/chapters/chapter01.md

```diff
@@ -4,48 +4,31 @@
-* [1.3 Operational Scope](#1.3)
+* [1.1 History of Anuket reference specifications](#1.1)
+* [1.2 How to participate](#1.2)
 
 <a name="1.1"></a>
-## 1.1 Who We Are
+## 1.1 History of Anuket reference specifications
 
-The founding members of the Cloud iNfrastructure Telco Task Force (CNTT) were AT&T, Vodafone, Verizon, Deutsche Telekom, Orange, and China Mobile. Soon thereafter, additional telco operator and vendor partner participants began to join the Task Force. CNTT reached its first major milestone when it gained sponsorship and support of the GSMA and Linux Foundation Networking in Summer 2019. As of June 2020, there were over thirty operators and partners (VNF suppliers, third-party integrators, hw/sw suppliers) in its member commmunity and these numbers continue to grow. Together the CNTT community is leading the industry in creating a common infrastructure reference platform in the form of reference model and reference architecture definitions to better support Network Function Virtualization (NFV) applications for the Telecom industry as a whole. The working group is collaborating with OPNFV and other open source communities to operationalize and support reference implementations and reference certification platforms. 
+Anuket reference specifications were founded as the Cloud iNfrastructure Telco Task Force (CNTT) by AT&T, Vodafone, Verizon, Deutsche Telekom, Orange, and China Mobile. Soon thereafter, additional telco operator and vendor partner participants began to join the Task Force. CNTT reached its first major milestone when it gained sponsorship and support of the GSMA and Linux Foundation Networking in Summer 2019. As of June 2020, there were over thirty operators and partners (VNF suppliers, third-party integrators, hw/sw suppliers) in its member commmunity and these numbers continue to grow. Together the CNTT community is leading the industry in creating a common infrastructure reference platform in the form of reference model and reference architecture definitions to better support Network Function Virtualization (NFV) applications for the Telecom industry as a whole. The working group was collaborating with OPNFV and other open source communities to operationalize and support reference implementations and reference certification platforms. 
 
-In order to establish governing guidelines and procedures by which the CNTT technical community may operate, the development of this document was undertaken by a CNTT workstream group and developed under the guidance of the CNTT Governance Steering Committee. 
+In the beginning of 2021 CNTT and OPNFV merged under the name [Anuket](https://anuket.io/) to leverage the synergies between the two projects. Former CNTT specifications are referred as Anuket reference specifications. Anuket operates under one Technical Oversight Commitee (TSC) and common governing rules documented in the Anuket Charter and in the Anuket Project Operations and Guidelines document. 
```

Comment:
Some of this is history and in a previous suggestion moved
For both the Charter and the Operations Guideline we should include the links

```suggestion
The Anuket project operates under a Technical Oversight Committee (TSC) and governing rules documented in the Anuket Charter and in the Anuket Project Operations and Guidelines document. 
```

Reply:
Project operations document is not yet approved.